spec/rubocop/cop/style/double_negation_spec.rb

```diff
@@ -116,6 +116,93 @@ def foo?
         end
       RUBY
     end
+
+    it 'does not register an offense for `!!` when return location and using `if`, `elsif`, and `else`' do
+      expect_no_offenses(<<~RUBY)
+        def foo?
+          if condition_foo?
+            !!foo
+          elsif condition_bar?
+            !!bar
+          else
+            !!baz
+          end
+        end
```

Comment:
It seems that false positives remain when the test will be updated as follows. The same is true for other tests.

```diff
          if condition_foo?
+           do_something
            !!foo
          elsif condition_bar?
+           do_something
            !!bar
          else
            !!baz
          end
        end
```

Reply:
I had overlooked that case. I'll fix it.